tensornetwork/network_operations.py

```diff
@@ -92,14 +93,14 @@ def transpose(node: BaseNode,
 
 def copy(nodes: Iterable[BaseNode],
          conjugate: bool = False) -> Tuple[dict, dict]:
-  """
+  """Return a copy of the given nodes.
 
-  Return a copy of the TensorNetwork.
   Args:
     nodes: An `Iterable` (Usually a `List` or `Set`) of `Nodes`.
     conjugate: Boolean. Whether to conjugate all of the nodes in the
-      `TensorNetwork` (useful for calculating norms and reduced density
-      matrices).
+        `TensorNetwork` (useful for calculating norms and reduced density
+        matrices).
+
   Returns:
     A tuple containing:
       node_dict: A dictionary mapping the nodes of the original
```

Comment:
Change wording to avoid use of "network".

Reply:
Done